apps/initialize-markets/src/Markets.ts

```diff
@@ -22,13 +22,16 @@ const SETTLE_ACCOUNTS_ABI = [
 ];
 
 const GRAPH_MAX_LIMIT = 1000;
-const graphUrl =
-  "https://api.studio.thegraph.com/query/36749/notional-v3-arbitrum/version/latest";
+
+export const defaultGraphEndpoints: Partial<Record<Network, (subgraphKey: string) => string>> = {
+    [Network.mainnet]: (subgraphKey: string) => `https://gateway-arbitrum.network.thegraph.com/api/${subgraphKey}/subgraphs/id/4oVxkMtN4cFepbiYrSKz1u6HWnJym435k5DQRAFt2vHW`,
```

Comment:
Can you pull the ids from the constants.ts file in case I have to change them later?